pkg/controller/namespace/namespace_controller.go

```diff
@@ -478,3 +482,17 @@ func deleteDeployments(expClient client.ExperimentalInterface, ns string) error
 	}
 	return nil
 }
+
+func deleteIngress(expClient client.ExperimentalInterface, ns string) error {
+	items, err := expClient.Ingress(ns).List(labels.Everything(), fields.Everything())
+	if err != nil {
+		return err
```

Comment:
Sorry, this is fine.   I couldn't tell easily on phone if this was in the
experimental if block.  Thanks!

On Monday, September 28, 2015, Prashanth B <notifications@github.com> wrote:

> In pkg/controller/namespace/namespace_controller.go
> <https://github.com/kubernetes/kubernetes/pull/14459#discussion_r40623670>
> :
>
> > @@ -478,3 +482,17 @@ func deleteDeployments(expClient client.ExperimentalInterface, ns string) error
> >  	}
> >  	return nil
> >  }
> > +
> > +func deleteIngress(expClient client.ExperimentalInterface, ns string) error {
> > +	items, err := expClient.Ingress(ns).List(labels.Everything(), fields.Everything())
> > +	if err != nil {
> > +		return err
>
> Hmm, it's either all or nothing, i mean the apiserver either has all
> experimental resources (exp=true) or doesn't. We're only in trouble if the
> controller manager is started with one of the exp flags and the apiserver
> isn't, in which case the autoscaler itself will fail. We have disjoint sets
> of flags that denote the same thing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14459/files#r40623670>.
>